File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/status-log_item.ts


import { IStatusLog_Item } from './istatus-log_item';

export class StatusLog_Item implements IStatusLog_Item {
  
	public TrainingEventStatusLogID: number = 0;
	public TrainingEventID: number = 0;
	public TrainingEventStatusID: number = 0;
	public TrainingEventStatus: string = "";
	public ReasonStatusChanged: string = "";
	public ModifiedByAppUserID: number = 0;
	public ModifiedDate: Date = new Date(0);
  
}

File: GTTS/INL.TrainingService.Models/StatusLog_Item.cs
using System;

namespace INL.TrainingService.Models
{
    public class StatusLog_Item : IStatusLog_Item
    {
        public long TrainingEventStatusLogID { get; set; }
        public long TrainingEventID { get; set; }
        public int TrainingEventStatusID { get; set; }
        public string TrainingEventStatus { get; set; }
        public string ReasonStatusChanged { get; set; }
        public int ModifiedByAppUserID { get; set; }
        public DateTime ModifiedDate { get; set; }
    }
}
